omnibus/config/software/datadog-agent.rb

```diff
@@ -41,7 +43,7 @@
         "Python2_ROOT_DIR" => "#{install_dir}/embedded",
         "Python3_ROOT_DIR" => "#{install_dir}/embedded",
         "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib -L#{install_dir}/embedded/lib",
-        "CGO_CFLAGS" => "-I#{install_dir}/embedded/include",
+        "CGO_CFLAGS" => "-I. -I#{install_dir}/embedded/include",
```

Comment:
Why do we need to add this include?

Reply:
This is a good question. I've already documented it in the commit message:

```
We added -I. before -I#{install_dir}/embedded/include in CGO_CFLAGS,
so the Go zstd package use its own zstd.h instead of using the one
provided by the newly added zstd software.
```

Before this change, we had to modify the name of the `zstd.h` file
in the Go [zstd](https://github.com/DataDog/zstd) package, so it doesn't conflicts with the newly installed 
`#{install_dir}/embedded/include/zstd.h`.

Reply:
Ah, thanks, I didn't notice the description in the commit message. Thanks!